tests/test_smart_proxy.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+
+@pytest.fixture
+def content_smart_proxy(api, organization):
+    content_proxy_id = '2'
+    content_proxy = api.resource('smart_proxies').call('show', {'id': content_proxy_id})
```

Comment:
@ekohl So I looked at picking the content proxy from the API... but the `mirror` property did not appear to be a capability and rather was a setting (which I didnt see in the API response).

Reply:
Ah right, we don't expose settings since they may contain secrets. For example, it can expose a username/password which we don't want to leak. I think we need a way to declare which settings we do expose (because they are safe).

Reply:
Can we add `mirror` into the set of capabilities? I tried looking at the smart-proxy code but was not as clear to me where to do that so it got the right value.

Reply:
https://github.com/theforeman/smart_proxy_pulp/blob/3eca0c86a3414fb5e8fdeb29dfd918d2a39f0ffa/lib/smart_proxy_pulp_plugin/pulpcore_plugin.rb#L14-L21 is where it's set.